data/css-line-grid-1.json

```diff
@@ -0,0 +1,3 @@
+{
+  "TR": "https://drafts.csswg.org/css-line-grid/"
```

Comment:
I think you need "css-line-grid-1" here because "css-line-grid" is not a proper shortname but an alias to "css-line-grid-1" (the code uses that shortname as identifier to retrieve information about the spec, and does not know anything about aliases)